### ISSUE TITLE
Incorrect commit URL

### ISSUE THREAD

**Issue by brainopia** [groupon/DotCi#168]
Incorrect commit URL is used to set a status of build.

The project is configured to use docker-compose. Maybe it's related. An example of such behavior:
https://ci.groupondev.com/job/RU/job/sms-service/4/console. 

It reports: `setting commit status on Github for `https://github.groupondev.com/api/v3/repos/RU/sms-service/commit/88f7a4270e710ea38bca18ca1251bc1980dbb1de

But this is an incorrect commit URL. It will return:

```json
{
  message: "Not Found",
  documentation_url: "https://developer.github.com/v3"
}
```

The URL should be slightly changed (a word `commit` should be replaced with `commits`) to be: 
https://github.groupondev.com/api/v3/repos/RU/sms-service/commits/88f7a4270e710ea38bca18ca1251bc1980dbb1de

This will return correct data:

```json
{
  sha: "88f7a4270e710ea38bca18ca1251bc1980dbb1de",
  ...
}
```

**Comment by suryagaddipati** [comment]
I think the link text is wrong but seems to do the right thing functionally(I will double check this). 


Keeping this open to remind me to fix the text.

**Comment by brainopia** [comment]
If functionality of commit status isn't broken then why is the commit https://github.groupondev.com/RU/sms-service/commit/88f7a4270e710ea38bca18ca1251bc1980dbb1de has no status?

**Comment by suryagaddipati** [comment]
@brainopia where are you expecting to see the status on github commit?

**Comment by brainopia** [comment]
@suryagaddipati I've expected to see it's on the page of the commit. Where else can I check the status of commit?

**Comment by suryagaddipati** [comment]
fixed this in `2.15.0`
